qa-niaid.planx-pla.net/portal/gitops.json

```diff
@@ -130,7 +130,8 @@
       "defaultOrderBy": ["title", "asc"],
       "buttons": [
         {
-          "type": "download"
+          "type": "download",
+          "overrideUrlField": "external_url"
```

Comment:
aw, we should have called `required_idp_field` => `requiredIdpField`

Reply:
do you think we could fix this inconsistency?

Reply:
ill make the change as well, just will need to remember to update the gitops after i push the portal changes

Reply:
thanks